src/components/Header/UrlField.tsx

```diff
@@ -46,6 +48,7 @@ export function UrlField({
   return (
     <EuiFieldText
       aria-label={URL_FIELD_LABEL}
+      fullWidth={fullWidth}
```

Comment:
Nit: the header is the only place we use`UrlField` so instead of having `fullWidth` as a prop we can simply remove it as a prop from `UrlField` itself and make the `EuiFieldText` always `fullWidth`.

Reply:
Fixed: https://github.com/elastic/synthetics-recorder/pull/178/commits/8a74b840967bd9f0992e96b48d0125335b3a98ce